testflight_invite.py

```diff
@@ -117,25 +128,62 @@ def service_key(self):
             raise ValueError('Unable to find iTunes Connect Service key')
         return matches.group(1)
 
+    def getGroups(self):
+        text = self.readData('https://itunesconnect.apple.com/testflight/v2/providers/{teamId}/apps/{appId}/groups'.format(
+            teamId=self.contentProviderId, appId=self.appId
+        ))
+        data = json.loads(text)
+        return data['data']
+
+    def getDefaultExternalGroupId(self):
+        if not hasattr(self, 'groupData'):
+            self.groupData = self.getGroups()
+        defaultExternalGroups = [g for g in self.groupData if g['isDefaultExternalGroup']]
```

Comment:
```
for group in self.groupData:
     if group['isDefaultExternalGroup']:
         return group['id']
return None
```
Seems like it would be more straightforward than logic below.